docker-compose.test.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+
+  mariadb:
+    build: docker/db
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: 'crms'
+    ports:
```

Comment:
Is there a reason to expose mysql when running tests?

Reply:
Worth trying without it --s econd-generation copy pasta. The original docker-compose has it for ease of use in trying schema changes (I think) but not needed here.

Reply:
Tests failed on `wait-for` (I think) with empty environment, using MYSQL_RANDOM_ROOT_PASSWORD subsequently allowed the tests to pass, so keeping that. A bit weird.